test/fixtures/game_teams.csv

```diff
@@ -0,0 +1,241 @@
+game_id,team_id,HoA,result,settled_in,head_coach,goals,shots,tackles,pim,powerPlayOpportunities,powerPlayGoals,faceOffWinPercentage,giveaways,takeaways
```

Comment:
These fixtures look good, we can likely remove some of the extra columns that are not needed, after reviewing all the methods.